docs/_documentation/advanced/mvxinteraction.md

```diff
@@ -91,11 +91,14 @@ public IMvxInteraction<YesNoQuestion> Interaction
     get => _interaction;
     set
     {
-        if (_interaction != null)
-            _interaction.Requested -= OnInteractionRequested;
+        if (value != null)
```

Comment:
The if should probably only be around the assigment and hooking up the event. Why shouldn't it be allowed to null out the interaction?